e2e/__tests__/__snapshots__/domDiffing.test.ts.snap

```diff
@@ -9,7 +9,7 @@ exports[`should work without error 1`] = `
 
     expect(received).toBe(expected) // Object.is equality
 
-    If it should pass with deep equality, replace \\"toBe\\" with \\"toStrictEqual\\"
+    If it should pass with deep equality, replace "toBe" with "toStrictEqual"
```

Comment:
here's an example where we didn't use `jest-snapshot-serializer-raw` and it's an improvement (unlike the places where we used it, which is now wrapped in `"`. IMO a bit uglier, but I think it makes sense to communicate it's a string)